token/core/cmd/pp/common/common.go

```diff
@@ -37,13 +46,23 @@ func GetMSPIdentity(entry string, mspID string) (view.Identity, error) {
 			return nil, errors.Errorf("invalid input [%s], expected <MSPConfigPath>:<MSPID> or <MSPConfigPath>", entry)
 		}
 	}
-	provider, err := x509.NewProvider(entries[0], mspID, nil)
+
+	// read certificate from entries[0]/signcerts
+	signcertDir := filepath.Join(entries[0], signcerts)
+	content, err := GetPemMaterialFromDir(signcertDir)
+	if err != nil {
+		return nil, errors.Wrapf(err, "failed to load certificates from %s", signcertDir)
+	}
```

Comment:
redundant error check?

Reply:
the folder might contain no files

Reply:
What I am saying is that we check the error twice! in line 52 and 58.

Reply:
my bad, sorry